### PR TITLE
Add centidegree unit

### PR DESCRIPTION
**Pull request by ziyan**
Centidegree unit is commonly used in the world of AGVs. Adding support for it can avoid using mysterious constants everywhere in the code.

### CODE REVIEW

File: include/openrave/units.h [hunk 1]
```diff
@@ -80,7 +80,8 @@ OPENRAVE_API TimeUnit GetTimeUnitFromString(const std::string& pTimeUnit, TimeUn
 enum AngleUnit : int8_t
 {
     AU_Radian = 0,
-    AU_Degree = 1
+    AU_Degree = 1,
+    AU_Centidegree = 2,
```
Comment (rdiankov) [inline include/openrave/units.h:84]: value should be `-2`
also need corresponding MR on schemaprofile repo to support this..

Comment (ziyan) [reply include/openrave/units.h]: Ok, I guess it's for comparing `AU_Degree > AU_Centidegree`

**Comment by ziyan** [comment]
Ping @rdiankov

**Comment by rdiankov** [comment]
thanks~

**Pull request merged**